extensions/wikia/JWPlayerTag/JWPlayerTag.i18n.php

```diff
@@ -0,0 +1,12 @@
+<?php
+$messages = [];
+
+$messages['en'] = [
+	'jwplayer-tag-desc' => 'JWPlayer Parser Tag for embedding video into articles',
```

Comment:
did you send translations to crowdin? is it release blocker?

Reply:
Nope, it's not :)